<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateStreamLineDialog</class>
 <widget class="QDialog" name="CreateStreamLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamLine</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Seed Number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Variable</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Direction</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="lengthHorizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="lengthDoubleSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>9999999999.999900817871094</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QSpinBox" name="seedNumSpinBox">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="directionComboBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>Forward</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Backward</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Both</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="typeComboBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Line With Arrow</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="vectorComboBox"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Integration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,3">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="initStepDoubleSpinBox">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999999999.999900817871094</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>MaxinumStep</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>MininumStep</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="maxStepDoubleSpinBox">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999999999.999900817871094</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="minStepDoubleSpinBox">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999999999.999900817871094</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>InitalStep</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="stepNumSpinBox">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Step Number</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sample Line</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2,1,1,2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="xStartPointLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>EndPoint</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="xEndPointLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>StartPoint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="zStartPointLineEdit"/>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="yStartPointLineEdit"/>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="yEndPointLineEdit"/>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="zEndPointLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateStreamLineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateStreamLineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
